src/lib.rs

```diff
@@ -82,6 +83,7 @@ pub use string::String;
 pub use vec::Vec;
 
 // NOTE this code was last ported from v0.4.1 of the indexmap crate
+mod deque;
```

Comment:
Maybe this comment should be moved ?

Reply:
:shrug: Seems like it was already wrong before, as I don't think it applied to `histbuf`. It seems very old anyway, I've removed it.